hubble/executor/parsers/__init__.py

```diff
@@ -126,4 +140,13 @@ def get_main_parser(parser=None):
         )
     )
 
+    set_hub_list_parser(
+        sp.add_parser(
+            'list',
+            help='display a list of local Jina executors',
+            description='Display a list of local Jina executors',
```

Comment:
```suggestion
            help='list your local Jina Executors',
            description='List your local Jina Executors',
```

Reply:
I will convert `Display a list of local Jina executors` to  `List your local Jina Executors`.